components/03-modules/minicart-product/minicart-product.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="minicart-product">
+    <div class="minicart-product__image">
+        {{ render '@image' productImage }}
+    </div>
+    <div class="minicart-product__info">
+        <h3 class="minicart-product__name">
+            {{ productName }}
+        </h3>
+        {{#if configurable }}
+            {{ render '@dropdown-list' attributes }}
+        {{/if}}
+        <div class="minicart-product__price">
+            {{ productPrice }}
+        </div>
+        <div class="minicart-product__bottom">
+            <div class="minicart-product__qty">
+                Qty: 32
```

Comment:
I think this text should be set in config

Reply:
done.